src/test/java/com/microsoft/sqlserver/testframework/AbstractTest.java

```diff
@@ -163,8 +163,7 @@ public static void setup() throws Exception {
             // no config file used
         }
 
-        connectionString = getConfiguredPropertyOrEnv(Constants.MSSQL_JDBC_TEST_CONNECTION_PROPERTIES);
-
+        connectionString = getConfiguredPropertyOrEnv(Constants.MSSQL_JDBC_TEST_CONNECTION_PROPERTIES); 
```

Comment:
This seems like an unneeded change to this file. I'd revert it so this file is unchanged in this PR.